apps-rendering/src/relatedContent.ts

```diff
@@ -53,27 +48,20 @@ const parseRelatedItemType = (content: Content): RelatedItemType => {
 };
 
 const parseHeaderImage = (content: Content): Image | undefined => {
```

Comment:
J ust a comment (not suggesting a change), but it's interesting we don't return an `Optional` here!

Reply:
👍 yeah we probably should do. I've avoided doing too much refactoring in one go in these PRs because they can spiral quickly and become hard to review!